### ISSUE TITLE
Carpet v 1.4.57 incompatibility

### ISSUE THREAD

**Issue by GunnyWaffle** [replaceitem/carpet-discarpet#21]
Seems there was a breaking change in carpet 1.4.57, where 1.4.56 operated fine.
I get this error now: (note that this is not server crashing, just console spamming)
```
[16:25:32] [Javacord - Central ExecutorService - 1/ERROR]: Unhandled exception in a listener thread for <discord server name here>!
java.lang.IllegalAccessError: class Discarpet.script.events.DiscordEvents tried to access private field carpet.script.CarpetEventServer$CallbackList.callList (Discarpet.script.events.DiscordEvents and carpet.script.CarpetEventServer$CallbackList are in unnamed module of loader net.fabricmc.loader.impl.launch.knot.KnotClassLoader @29ee9faa)
        at Discarpet.script.events.DiscordEvents.callHandlerInBotApps(DiscordEvents.java:135) ~[discarpet-1.18-1.1.0.jar:?]
        at Discarpet.script.events.DiscordEvents$1.onDiscordMessage(DiscordEvents.java:46) ~[discarpet-1.18-1.1.0.jar:?]
        at Discarpet.Bot.lambda$new$2(Bot.java:52) ~[discarpet-1.18-1.1.0.jar:?]
        at org.javacord.core.util.event.EventDispatcher.lambda$dispatchMessageCreateEvent$122(EventDispatcher.java:4371) ~[org_javacord_javacord-core-3.3.2-c871f0db7a27cbab.jar:?]
        at org.javacord.core.util.event.EventDispatcherBase.lambda$dispatchEvent$12(EventDispatcherBase.java:209) ~[org_javacord_javacord-core-3.3.2-c871f0db7a27cbab.jar:?]
        at org.javacord.core.util.event.EventDispatcherBase.lambda$checkRunningListenersAndStartIfPossible$20(EventDispatcherBase.java:277) ~[org_javacord_javacord-core-3.3.2-c871f0db7a27cbab.jar:?]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539) [?:?]
        at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) [?:?]
        at java.lang.Thread.run(Thread.java:833) [?:?]
```
Let me know if you need anything else!

**Comment by cherie535** [comment]
ah, yes, I was just going to google this.  Me too on the above

**Comment by replaceitem** [comment]
Fixed by 5988e3cc8a3c3790efdab4064a77458190dfd0b6
